catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx

```diff
@@ -151,6 +151,14 @@ const SearchLayout: FunctionComponent = ({ children }) => {
             return false;
           }}
         />
+        <h1 className="visually-hidden">
+          {/* Uppercases the first letter */}
+          {`${
+            currentSearchCategory[0].toUpperCase() +
+            currentSearchCategory.substring(1)
+          } search page`}
```

Comment:
You can use https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/utils/grammar.ts#L1-L6